app/presenters/finder_presenter.rb

```diff
@@ -186,6 +187,15 @@ def all_content_finder?
     self.content_item['content_id'] == 'dd395436-9b40-41f3-8157-740a453ac972'
   end
 
+  def topic_finder?
```

Comment:
I would use or move the `i_am_a_topic_page_finder` method here.

```
def i_am_a_topic_page_finder
   values[:topic].present? && Services.registries.all['full_topic_taxonomy'].taxonomy_registry.taxonomy.key? values[:topic]
end
```

Reply:
The `i_am_a_topic_finder` seems to use `?parent=` as opposed to `?topic=` as a parameter in the URL - so I'm hesitant to either copy or alter it, as it seems to have subtly different usage.